projects/plugins/jetpack/modules/wordads/class-wordads.php

```diff
@@ -291,16 +291,8 @@ private function insert_adcode() {
 			if ( self::is_amp() ) {
 				add_filter( 'the_content', array( $this, 'insert_header_ad_amp' ) );
 			} else {
-				switch ( get_stylesheet() ) {
-					case 'twentyseventeen':
-					case 'twentyfifteen':
-					case 'twentyfourteen':
-						add_action( 'wp_footer', array( $this, 'insert_header_ad_special' ) );
-						break;
-					default:
-						add_action( 'wp_head', array( $this, 'insert_header_ad' ), 100 );
-						break;
-				}
```

Comment:
Should we worry about getting rid about those custom implementations for default themes?

Reply:
This switch statement essentially gets moved into the `insert_header_ad` method: https://github.com/Automattic/jetpack/pull/22016/files#diff-6a0a715a9c07ca5427d04aca1f28877c6185aba3d94f68988d5bc9f38ff771c7R493-L517